### ISSUE TITLE
Fix Immersive styling issues

### ISSUE THREAD

**Issue by petternitter** [guardian/dotcom-rendering#6074]
Following the release of the Immersive format, a few styling issues have been reported that we should look to resolve.

- H2 headlines are displaying in incorrect font size, should be corrected to 34 px
- The top tag background breaks on Android but appears correctly on iOS

Screenshots below for further context

**H2 headlines**
![Image](https://user-images.githubusercontent.com/55494673/192759322-ea536c46-e7c2-4012-9bfc-dae39f890eb5.png)

**Top tag**
![Image](https://user-images.githubusercontent.com/55494673/192759563-491838a3-ce92-40cd-82c2-95b1752fe989.png)

**Comment by ajbreuer** [comment]
@petternitter to answer the questions...

H2's should be 34px
The tag only appears to break on Android, is fine on IOS
Re the drop caps, there seems to be a disparity between DCR and AR. We have isolated that the kind of return has an influence (soft v hard). It feels like a chat with engineers on this would be good as there is also the complexity of line length for the bold text, a short line gets rid of the drop cap. Feels like we need to finesse the rules. And perhaps, as there is an embed (that styles the red buttons) on the climate change article, could this also be causing problems...

**Comment by petternitter** [comment]
Super, thanks, Alex. I'll add info on two first issues to this ticket and then break out the drop cap issue into a separate ticket we can move forward in collaboration with Dotcom.

**Comment by petternitter** [comment]
@JamieB-gu Not sure what effort would be involved in resolving the 2 issues that remain in this ticket but obviously great if we can evidence the power of AR by resolving those pretty swiftly

**Comment by JamieB-gu** [comment]
I've addressed the first two points in the above PRs.

> The tag only appears to break on Android, is fine on IOS

@ajbreuer it looks like it was only wrapping on specific breakpoints, so I think it would have wrapped on devices with very small screens?

> Re the drop caps, there seems to be a disparity between DCR and AR.

There are a couple of known issues on DCR in #4813 and #4915, and an open question for both platforms in #5967.

> We have isolated that the kind of return has an influence (soft v hard).

I wasn't aware of the disparity with returns! Is there an open issue anywhere that captures that?

> It feels like a chat with engineers on this would be good as there is also the complexity of line length for the bold text, a short line gets rid of the drop cap. Feels like we need to finesse the rules.

👍 I think the most up-to-date version of the rules can be found described in #5967. These were what we implemented, based on the Editorial documentation on this, when drop-caps were implemented in AR: #4674. I think DCR follows these too, with the exception of the known issues mentioned above.

There's nothing specified about what to do when some of the paragraph text is wrapped in a `<strong>` (which is used to create the bold effect).
